Method signature mismatch when compiling with DECO_CALC_DEBUG

Commit 8e21a6565351 by @atdotde  updates dump_tissues in dive.h to: 

extern void dump_tissues(struct deco_state *ds);

However, it doesn't update the reference in profile.c line 1150.

Build fails: 

/home/martin/src/subsurface/core/profile.c: In function ‘calculate_deco_information’:
/home/martin/src/subsurface/core/profile.c:1150:2: error: too few arguments to function ‘dump_tissues’
  dump_tissues();
  ^~~~~~~~~~~~
In file included from /home/martin/src/subsurface/core/profile.c:10:0:
/home/martin/src/subsurface/core/dive.h:905:13: note: declared here
 extern void dump_tissues(struct deco_state *ds);
             ^~~~~~~~~~~~
core/CMakeFiles/subsurface_corelib.dir/build.make:710: recipe for target 'core/CMakeFiles/subsurface_corelib.dir/profile.c.o' failed
make[2]: *** [core/CMakeFiles/subsurface_corelib.dir/profile.c.o] Error 1
CMakeFiles/Makefile2:473: recipe for target 'core/CMakeFiles/subsurface_corelib.dir/all' failed
make[1]: *** [core/CMakeFiles/subsurface_corelib.dir/all] Error 2
Makefile:129: recipe for target 'all' failed

--- a/core/divelist.c
+++ b/core/divelist.c
@@ -572,7 +572,7 @@ int init_decompression(struct deco_state *ds, struct dive *dive)
 			deco_init = true;
 #if DECO_CALC_DEBUG & 2
 			printf("Tissues after init:\n");
-			dump_tissues();
+			dump_tissues(ds);
 #endif
 		}
 		else {

--- a/core/profile.c
+++ b/core/profile.c
@@ -1147,7 +1147,7 @@ void calculate_deco_information(struct deco_state *ds, struct deco_state *planne
 	}
 	free(cache_data_initial);
 #if DECO_CALC_DEBUG & 1
-	dump_tissues();
+	dump_tissues(ds);
 #endif
 	unlock_planner();
 }

